tds_test.go

```diff
@@ -584,6 +585,32 @@ func TestBadHost(t *testing.T) {
 	testConnectionBad(t, params.URL().String())
 }
 
+func TestSqlBrowserNotUsedIfPortSpecified(t *testing.T) {
```

Comment:
it would be possible to write a test that enumerates the local SQL instances to find the port to set for a positive test case. Such a case would run fine in the appveyor test run and skip itself if it finds no instances.
See https://www.appveyor.com/docs/services-databases/
But it's not a big deal I think.